vspace/src/test/java/edu/asu/diging/vspace/core/services/impl/SequenceManagerTest.java

```diff
@@ -0,0 +1,90 @@
+package edu.asu.diging.vspace.core.services.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import edu.asu.diging.vspace.core.data.SequenceRepository;
+import edu.asu.diging.vspace.core.factory.impl.SequenceFactory;
+import edu.asu.diging.vspace.core.model.ISequence;
+import edu.asu.diging.vspace.core.model.ISlide;
+import edu.asu.diging.vspace.core.model.impl.Module;
+import edu.asu.diging.vspace.core.model.impl.Sequence;
+import edu.asu.diging.vspace.web.staff.forms.SequenceForm;
+
+public class SequenceManagerTest {
+
+    @Mock
+    private SequenceRepository mockSequenceRepo;
+
+    @Mock
+    private ModuleManager moduleManager;
+    
+    @Mock
+    private SlideManager slideManager;
+    
+    @Mock
+    private SequenceFactory sequenceFactory;
+    
+    @InjectMocks
+    private SequenceManager sequenceToTest = new SequenceManager();
```

Comment:
you're not testing a sequence but the sequenceManager, so your variable name should reflect that.